ui-components/src/helpers/filters.ts

```diff
@@ -28,15 +29,18 @@ function getComparisonForFilter(filterKey: ListingFilterKeys) {
   }
 }
 
-export function encodeToBackendFilterString(filterParams: ListingFilterParams) {
-  let queryString = ""
+export function encodeToBackendFilterArray(filterParams: ListingFilterParams) {
```

Comment:
I think it's out of scope for this merge PR, but it's worth considering writing some helper functions that build a full listing querystring, including the limit and pages, so we don't have to use qs to encode the output of this everywhere.

Reply:
Sounds good.